src/TestSuite/IntegrationTestCase.php

```diff
@@ -243,6 +259,17 @@ public function enableCsrfToken()
     }
 
     /**
+     * Calling this method will re-store flash messages into the test session
+     * after being removed by the FlashHelper
+     *
+     * @return void
+     */
+    public function enableRememberFlashMessages()
```

Comment:
``enableRetainFlashMessages()`` or  ``retainFlashMessages()``?

"Remember" is quite human.